sensor_msgs/Image Image
---
geometry_msgs/Pose Pose
